index.html

```diff
@@ -1,156 +1,328 @@
+          <button class="see-project-button">Get my resume</button>
+        </div>
+        <ul class="about-myself-skills principal-margin">
+          <li class="arrow-skills">
+            <p>Languages</p>
+            <i class="fa fa-angle-down arrow-color"></i>
+          </li>
           <ul class="tech-languages">
-            <li><img src="Images_Background/JS-logo.svg" alt="JavaScript logo"><p class="language-name">JavaScript</p></li>
-            <li><img src="Images_Background/html-logo.svg" alt="HTML logo"><p class="language-name">HTML</p></li>
-            <li><img src="Images_Background/css-logo.svg" alt="CSS logo"><p class="language-name">CSS</p></li>
+            <li>
+              <img src="Images_Background/JS-logo.svg" alt="JavaScript logo" />
+              <p class="language-name">JavaScript</p>
+            </li>
+            <li>
+              <img src="Images_Background/html-logo.svg" alt="HTML logo" />
+              <p class="language-name">HTML</p>
+            </li>
+            <li>
+              <img src="Images_Background/css-logo.svg" alt="CSS logo" />
+              <p class="language-name">CSS</p>
+            </li>
           </ul>
-          <li class="bottom-line-language arrow-skills"><p>Frameworks</p><i class="fa fa-angle-down arrow-color"></i></li>
-          <li class="bottom-line-language arrow-skills"><p>Skills</p><i class="fa fa-angle-down arrow-color"></i></li>
-      </ul>
-    </section>
-  </main>  
-</body>
-</html>
\ No newline at end of file
+          <li class="bottom-line-language arrow-skills">
+            <p>Frameworks</p>
+            <i class="fa fa-angle-down arrow-color"></i>
+          </li>
+          <li class="bottom-line-language arrow-skills">
+            <p>Skills</p>
+            <i class="fa fa-angle-down arrow-color"></i>
+          </li>
+        </ul>
+      </section>
+
+      <!------------------------>
+      <!-- CONTACT ME SECTION -->
+      <!------------------------>
+
+      <section id="contact-me">
+        <h2 class="contact-me-title">Contact me</h2>
+        <p class="contact-me-paragraph principal-margin">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it!
+        </p>
+        <form action="https://formspree.io/f/mknayrpk" method="post" class="principal-margin">
+          <input
+            type="text"
+            id="name"
+            class="form-input form-input-width"
+            name="user-name"
+            maxlength="30"
+            placeholder="Hernán Güemes"
+            required
+          />
+          <input
+            type="email"
+            class="form-input form-input-width"
+            id="mail"
+            name="user-email"
+            placeholder="hernanguemes@hotmail.com"
+            required
+          />
+          <textarea
+            name="message"
+            id="message"
```

Comment:
- Kindly make Sure to follow your last Reviewer Feedback [here](https://github.com/HFG43/Portfolio_Mobile_First/pull/3#discussion_r1182946021) and stick to the Figma design just as requested. I think it would be better to change the placeholder text.

**Your Design**
![235757789-544ce0c4-f8f0-42d7-8c52-6f8ae9f5ea64](https://user-images.githubusercontent.com/97834160/235774767-a86e5e77-171f-47ff-bf9f-e6938fa9541c.png)
**Figma Design**
![235757723-b5b32023-34fe-4ef6-9a66-b37871b4201c](https://user-images.githubusercontent.com/97834160/235774807-5b6b4755-ce4c-424e-9ec1-b184ec234588.png)

Reply:
@GedeonTS Hi! Many thanks for your feedback. Please let me know if I can please call you through zoom one moment, if that's fine for you